pkg/hintrunner/zero/zerohint_signatures.go

```diff
@@ -50,3 +53,81 @@ func createVerifyECDSASignatureHinter(resolver hintReferenceResolver) (hinter.Hi
 	}
 	return newVerifyECDSASignatureHinter(ecdsaPtr, signature_r, signature_s), nil
 }
+
+func newGetPointFromXHinter(xCube, v hinter.ResOperander) hinter.Hinter {
+	return &GenericZeroHinter{
+		Name: "VerifyECDSASignature",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
+			//> x_cube_int = pack(ids.x_cube, PRIME) % SECP_P
+			//> y_square_int = (x_cube_int + ids.BETA) % SECP_P
+			//> y = pow(y_square_int, (SECP_P + 1) // 4, SECP_P)
+			//> if ids.v % 2 == y % 2:
+			//>	 value = y
+			//> else:
+			//>	 value = (-y) % SECP_P
+			xCubeAddr, err := xCube.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			xCubeMemoryValues, err := hinter.GetConsecutiveValues(vm, xCubeAddr, 3)
+			if err != nil {
+				return err
+			}
+			v, err := hinter.ResolveAsFelt(vm, v)
+			if err != nil {
+				return err
+			}
+
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
+			secpBig, _ := secp_utils.GetSecPBig()
```

Comment:
The idea is to not check for `err` because we know its always successful?

Reply:
yes